INSTALL.md

```diff
@@ -0,0 +1,117 @@
+# 📝 Installation for GCP Scanner
+
+## Install basic dependencies
+
+**Linux** is the preferred operating system to use while contributing to GCP Scanner. If you're using Windows, we recommend setting up [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```

Comment:
Could you open a bug to add the same description but for MacOS?

Reply:
Done :+1: 
See #125